logstash-core/lib/logstash/pipeline.rb

```diff
@@ -379,12 +379,16 @@ def plugin(plugin_type, name, *args)
     klass = LogStash::Plugin.lookup(plugin_type, name)
 
     if plugin_type == "output"
-      LogStash::OutputDelegator.new(@logger, klass, *args)
+      LogStash::OutputDelegator.new(@logger, klass, default_output_workers, *args)
     else
       klass.new(*args)
     end
   end
 
+  def default_output_workers
+    @settings[:pipeline_workers] || @settings[:default_pipeline_workers]
+  end
```

Comment:
Do we really need to have a check for this? The `DEFAULT_SETTINGS` already have a default settings configured? 
I think the fallback to defaults should be done in one place https://github.com/elastic/logstash/pull/4391/files#diff-bc1e9ea81caa54ea0d56abf4415fdf00R34 ? WDYT?